src/Pages/CreateQuestion.tsx

```diff
@@ -1,35 +1,97 @@
 import styled from '@emotion/styled'
 import { FillBtn } from '../Components/basic/button/Button'
-import { TextInput } from '../Components/basic/InputBox'
-import { Label } from '../Components/basic/Label'
-import { CategoryInput } from '../Components/CategoryInput'
 import { TextEditor } from '../Components/TextEditor'
+import { useState } from 'react';
```

Comment:
did you setup prettier and eslint on your editor? the rule is to avoid adding semicolons.

Reply:
> did you setup prettier and eslint on your editor? the rule is to avoid adding semicolons.

I have updated the styling now :)